tests/test_utils_log.py

```diff
@@ -100,21 +99,19 @@ def test_filtered_out_level(self):
         assert self.crawler.stats.get_value("log_count/INFO") is None
 
 
-class StreamLoggerTest(unittest.TestCase):
-    def setUp(self):
```

Comment:
pytest overrides `sys.stdout` after the fixtures etc. run, so merging the code into the test method was the easiest way to reimplement this